src/github.com/getlantern/flashlight/client/config.go

```diff
@@ -20,6 +21,10 @@ type ClientConfig struct {
 	ChainedServers map[string]*ChainedServerInfo
 	Peers          map[string]*nattywad.ServerPeer // keyed to peer id (e.g. XMPP JID)
 	MasqueradeSets map[string][]*fronted.Masquerade
+
+	OpenUi    bool              // Flag specifying whether or not the UI should be automatically open in the browser
+	UiAddr    string            // UI HTTP server address
```

Comment:
Why is the UI stuff under client?  At some point, give-mode (i.e. server) flashlight will also use the UI, right?